org.remmina.Remmina.json

```diff
@@ -0,0 +1,494 @@
+{
+    "app-id": "org.remmina.Remmina",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "command": "remmina",
+    "rename-icon": "remmina",
+    "desktop-file-name-prefix": "(Flatpak) ",
```

Comment:
Unnecessary

Reply:
@amtlib-dot-dll This one =)

Reply:
https://github.com/amtlib-dot-dll/flathub/commit/aecd39712ea6e4c75acef535dcf37c2584da8598 Thank you for notifying :joy: